tests/integration/records/single_db/test_records_numeric.py

```diff
@@ -88,7 +88,9 @@ def test_numeric_database_columns_created(self):
             'redshift': 'bluelabs',
             'bigquery': 'bigquery',
             'vertica': 'vertica',
+            # TODO: Test postgres and see if this is right - is this test valid if we use pandas?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker